packages/relay/src/RelayServer.ts

```diff
@@ -521,10 +521,6 @@ returnValue        | ${viewRelayCallRet.returnValue}
 
     this.chainId = this.contractInteractor.chainId
     this.networkId = this.contractInteractor.getNetworkId()
-    if (this.config.devMode && (this.chainId < 1000 || this.networkId < 1000)) {
-      this.logger.error('Don\'t use real network\'s chainId & networkId while in devMode.')
-      process.exit(-1)
-    }
```

Comment:
there is this code:
```
if (devMode) {
    if (fs.existsSync(`${workdir}/${TXSTORE_FILENAME}`)) {
      fs.unlinkSync(`${workdir}/${TXSTORE_FILENAME}`)
    }
```

This is the only place I've seen with 'devMode' being used - and this is before the check.

Do we even need this parameter?

Reply:
I think some tests still rely on removing these files, but I believe we can remove it and remove those files in the tests themselves and not from runServer...